verify/trust/trust.go

```diff
@@ -160,9 +183,16 @@ func (n *SimpleHTTPSGetter) Get(url string) ([]byte, error) {
 	return body, nil
 }
 
+var (
+	_ = HTTPSGetter(&SimpleHTTPSGetter{})
+	_ = ContextHTTPSGetter(&SimpleHTTPSGetter{})
+)
+
```

Comment:
Please move to a test. These have no business being constructed in the main library.